src/test/common/steps/confirm-details.js

```diff
@@ -33,6 +42,13 @@ async function checkAllPageContentIsPresentAndCorrect () {
   expect(panelTitle.toString().trim()).to.be.equal('Application complete', 'expected confirmation header to be correct')
 }
 
+async function checkExistingClaimUpdatedTextIsPresent () {
```

Comment:
I'm guessing we have no different content at all yet? If so, might be worthwhile adding a TODO with a Jira number in it for reference.

Reply:
Done.